### PR TITLE
Enable Avalanche for Swaps

### PR DESCRIPTION
**Pull request by dan437**
## Explanation
This PR enables Avalanche to be used in Swaps. Most of the work for this is done on backend, so frontend just needs this small change.

## Manual testing steps
- Start the extension with `SWAPS_USE_DEV_APIS=1 yarn start`
- Add Avalanche into Networks in MM
- Test out Swaps

### CODE REVIEW

**Review by danjm — APPROVED** [review]

File: lavamoat/browserify/beta/policy.json [hunk 1]
```diff
@@ -1839,16 +1839,6 @@
         "safe-event-emitter": true
       }
     },
-    "eth-eip712-util-browser": {
```
Comment (danjm) [inline lavamoat/browserify/beta/policy.json:1842]: Any reason why these policy changes are in here?

**Review by danjm — CHANGES_REQUESTED** [review]

**Comment by darkwing** [comment]
Marking as DNM until the service is ready.

File: lavamoat/browserify/beta/policy.json (hunk 1 above)
Comment (dan437) [reply lavamoat/browserify/beta/policy.json:1842]: When I ran the extension, it was showing a white screen. When I opened the Console tab from background.html, there was a JS error. After running `yarn lavamoat:auto` and `yarn start` it all works well. That's why I committed the changes, so people don't need run `yarn lavamoat:auto` and wait for a long time until it's done.

If there is a different way to handle this, let me please know.

Comment (dan437) [reply lavamoat/browserify/beta/policy.json]: Update: maybe I had that LavaMoat issue after I switched from another local branch. I've removed the policies from this PR and it seems that locally it's loading correctly now.

File: shared/constants/swaps.js [hunk 2]
```diff
@@ -83,12 +96,14 @@ export const WBNB_CONTRACT_ADDRESS =
...
   '0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270';
+export const WAVAX_CONTRACT_ADDRESS =
+  '0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7';
 
 const SWAPS_TESTNET_CHAIN_ID = '0x539';
 
 export const SWAPS_API_V2_BASE_URL = 'https://api2.metaswap.codefi.network';
 export const SWAPS_DEV_API_V2_BASE_URL =
-  'https://api2.metaswap-dev.codefi.network';
+  'https://swap.metaswap-dev.codefi.network';
```
Comment (darkwing) [inline shared/constants/swaps.js:106]: Is this an intended change for this PR?

**Review by brad-decker — DISMISSED** [review]

**Review by brad-decker — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Review by darkwing — APPROVED** [review]
Just tested and works great!  Let's goooooo!

**Pull request merged**